rosdep/python.yaml

```diff
@@ -8226,6 +8226,13 @@ python3-smc-pip:
   ubuntu:
     pip:
       packages: [smc]
+python3-socketio:
+  debian:
+    bullseye: [python3-socketio]
```

Comment:
Similar to cottsay's suggestion for forward-proofing the definition on Ubuntu we should do the same here.
```suggestion
    '*': [python3-socketio]
    buster: null
```

Reply:
Taking my own suggestion in order to move this forward.